docs/reference/pip_install.rst

```diff
@@ -21,9 +21,52 @@ Overview
 
 Pip install has several stages:
 
-1. Resolve dependencies. What will be installed is determined here.
-2. Build wheels. All the dependencies that can be are built into wheels.
-3. Install the packages (and uninstall anything being upgraded/replaced).
+1. Identify the base requirements. The user supplied arguments are processed
+   here.
+2. Resolve dependencies. What will be installed is determined here.
+3. Build wheels. All the dependencies that can be are built into wheels.
```

Comment:
Might be worth noting this can be disabled with `--no-binary` and for point 2. `--no-deps` with links to the options description ?
But maybe that's not the best place ?